sql/moz-fx-data-shared-prod/braze_derived/users_v1/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH
+  exclusions AS (
+  SELECT
+    e.email_id,
+    e.primary_email AS email,
+    e.mailing_country,
+    CASE
+      WHEN e.double_opt_in = TRUE THEN 'opted_in'
+      WHEN e.double_opt_in = FALSE
+    AND e.has_opted_out_of_email = FALSE THEN 'subscribed'
+    ELSE
+    'unsubscribed'
+  END
+    AS email_subscribe,
+    e.basket_token,
+    e.email_lang,
+    e.create_timestamp AS create_timestamp,
+    e.update_timestamp AS update_timestamp,
+    @submission_date AS last_modified_timestamp,
+    DATE(@submission_date) AS last_modified_date
+  FROM
+    `moz-fx-data-shared-prod.ctms_braze.ctms_emails` e
+  LEFT JOIN
+    `moz-fx-data-shared-prod.braze_derived.suppressions` s
+  ON
+    LOWER(e.primary_email) = s.email
+  WHERE
+    s.email IS NULL
+    AND update_timestamp > (
+    SELECT
+      MAX(last_modified_timestamp)
+    FROM
+      `moz-fx-data-shared-prod.braze_derived.users_v1` ) )
+SELECT
+  email_id,
+  email,
+  mailing_country,
+  email_subscribe,
+  basket_token,
+  email_lang,
+  create_timestamp,
+  update_timestamp,
+  @submission_date AS last_modified_timestamp,
```

Comment:
Thinking ahead to a future where we _may_ want to sync data into Braze more frequently than 1x per day, does `@submission_date` resolve to a datetime, or just a date?

If we sync data into Braze more than once per day, it'll be necessary to store the full last modified timestamp so that we can avoid syncing the same records into Braze more than once.

Reply:
yes, submission_date is actually a timestamp :)

Reply:
gotta love the logic behind that one :)